test/spicepods/tpch/sf1/accelerated/append/file_cayenne_file.yaml

```
@@ -0,0 +1,41 @@
+version: v1
+kind: Spicepod
+name: file_cayenne_file_append
```

```suggestion
name: file[parquet]-cayenne[file]-append
```

these spicepod names need updating to match convention

---

@peasee - good point, thank you, updated!